Whiskey/Functions/Install-WhiskeyDotnetSdk.ps1

```diff
@@ -0,0 +1,99 @@
+
+function Install-WhiskeyDotnetSdk
+{
+    <#
+    .SYNOPSIS
+    Installs the .NET Core SDK tooling.
+
+    .DESCRIPTION
+    The `Install-WhiskeyDotnetSdk` function will install the .NET Core SDK tools and return the path to the installed `dotnet.exe` command. If you specify the `SearchExisting` switch then the function will first look for any globally installed .NET Core SDK's with the desired version already installed. If one is found, then install is skipped and the path to the global install is returned. The function uses the `dotnet-install.ps1` script from the [dotnet-cli](https://github.com/dotnet/cli) GitHub repository to download and install the SDK.
+
+    .EXAMPLE
+    Install-WhiskeyDotnetSdk -InstallRoot 'C:\Build\.dotnet' -Version '2.1.4'
+
+    Demonstrates installing .NET Core SDK version 2.1.4 to the 'C:\Build\.dotnet' directory. After install the function will return the path 'C:\Build\.dotnet\dotnet.exe'.
+
+    .EXAMPLE
+    Install-WhiskeyDotnetSdk -InstallRoot 'C:\Build\.dotnet' -Version '2.1.4' -SearchExisting
+
+    Demonstrates searching for an existing global install of the .NET Core SDK version '2.1.4'. If not found globally, the SDK will be installed to 'C:\Build\.dotnet'.
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]
+        # Directory where the .NET Core SDK will be installed.
+        $InstallRoot,
+
+        [Parameter(Mandatory=$true)]
+        [string]
+        # Version of the .NET Core SDK to install.
+        $Version,
+
+        [switch]
+        # Search for the desired version from existing global installs of the .NET Core SDK. If found, the install is skipped and the path to the global install is returned.
+        $SearchExisting

```

Comment:
I renamed this parameter many times, not quite able to come up with the best name. I started with `Global` but thought that wasn't entirely accurate since the function never installs the Sdk globally, it just searches for a global install before installing *locally* to the given directory (i.e. it doesn't add the install directory to the `Path` environment variable).

Are we OK with still naming it `Global` given the above?

Reply:
I reviewed the [list of standard PowerShell parameter names/types](https://msdn.microsoft.com/en-us/library/dd878352.aspx) and the closest I could find is `Scope`, but that's a string, not a switch. Since we both initially went with `Global`, let's use that. Hopefully, that's an indication that's what other developer will think/reach for.